e2e/provision/init.sh

```diff
@@ -70,14 +70,22 @@ REPO=${NEPHIO_REPO:-$(get_metadata nephio-test-infra-repo "https://github.com/ne
 BRANCH=${NEPHIO_BRANCH:-$(get_metadata nephio-test-infra-branch "main")}
 NEPHIO_USER=${NEPHIO_USER:-$(get_metadata nephio-user "${USER:-ubuntu}")}
 NEPHIO_CATALOG_REPO_URI=${NEPHIO_CATALOG_REPO_URI:-$(get_metadata nephio-catalog-repo-uri "https://github.com/nephio-project/catalog.git")}
-export ANSIBLE_CMD_EXTRA_VAR_LIST="nephio_catalog_repo_uri='$NEPHIO_CATALOG_REPO_URI'"
+K8S_CONTEXT=${K8S_CONTEXT:-"kind-kind"}
+K8S_VERSION=${K8S_VERSION:-"v1.29.2"}
+export ANSIBLE_CMD_EXTRA_VAR_LIST='{ "nephio_catalog_repo_uri": "'${NEPHIO_CATALOG_REPO_URI}'", "k8s": { "context" : "'${K8S_CONTEXT}'", "version" : "'$K8S_VERSION'" } }'
```

Comment:
@vjayaramrh ANSIBLE_CMD_EXTRA_VAR_LIST variable can be provided by the user, so this overrides its value

```suggestion
export ANSIBLE_CMD_EXTRA_VAR_LIST="${ANSIBLE_CMD_EXTRA_VAR_LIST:-'{ "nephio_catalog_repo_uri": "'${NEPHIO_CATALOG_REPO_URI}'", "k8s": { "context" : "'${K8S_CONTEXT}'", "version" : "'$K8S_VERSION'" } }'}"
```

Reply:
Good catch, thanks

Reply:
The original code also did not account for user provided value for ANSIBLE_CMD_EXTRA_VAR_LIST. We should treat user provided input for this separately as we need to test out other cases.
The good thing is I have tested out case where multiple `--extra-vars` can be supplied to a playbook and this can be used for future implementation.

Reply:
This [env variable is used by Vagrant](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/Vagrantfile#L61), so overriding its value will result in a misfunctioning. Could you find a way to avoid that misfunctioning?

Reply:
@electrocucaracha Thanks for the comment, I plan to open an issue to be fixed as part of a later PR. Would that be OK?. I noticed that you have 'https://github.com/nephio-project/nephio-catalog-packages.git in the link above that is not valid. Is that by design or should it have been https://github.com/nephio-project/catalog.git?

Reply:
FYI I have opened an issue to address this post R3 at https://github.com/nephio-project/nephio/issues/777